ImageKitSample/ImageKitSample.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Does it have anything to do with the SDK's requirement to work in .NET Standard 2.0+, .NET Core 2.0+, and .NET Framework 4.6.1+. If yes, should we change this value?